ts/src/bingx.ts

```diff
@@ -0,0 +1,2342 @@
+        //
+        //     {
+        //         "symbol": "BTC-USDT",
+        //         "positionId": "12345678",
+        //         "positionSide": "LONG",
+        //         "isolated": true,
+        //         "positionAmt": "123.33",
+        //         "availableAmt": "128.99",
+        //         "unrealizedProfit": "1.22",
+        //         "realisedProfit": "8.1",
+        //         "initialMargin": "123.33",
+        //         "avgPrice": "2.2",
+        //         "leverage": 10,
+        //     }
+        //
+        const marketId = this.safeString (position, 'symbol');
+        const isolated = this.safeValue (position, 'isolated');
+        const marginMode = isolated ? 'isolated' : 'cross';
+        return this.safePosition ({
+            'info': position,
+            'id': this.safeString (position, 'positionId'),
+            'symbol': this.safeSymbol (marketId, market, '-', 'swap'),
+            'notional': this.safeString (position, 'positionAmt'),
+            'marginMode': marginMode,
+            'liquidationPrice': undefined,
+            'entryPrice': this.safeNumber (position, 'avgPrice'),
+            'unrealizedPnl': this.safeNumber (position, 'unrealizedProfit'),
+            'percentage': undefined,
+            'contracts': undefined,
+            'contractSize': undefined,
+            'markPrice': undefined,
+            'lastPrice': undefined,
+            'side': this.safeStringLower (position, 'positionSide'),
+            'hedged': undefined,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'lastUpdateTimestamp': undefined,
+            'maintenanceMargin': undefined,
+            'maintenanceMarginPercentage': undefined,
+            'collateral': this.safeString (position, 'positionAmt'),
+            'initialMargin': this.safeNumber (position, 'initialMargin'),
+            'initialMarginPercentage': undefined,
+            'leverage': this.safeNumber (position, 'leverage'),
+            'marginRatio': undefined,
+        });
+    }
+
+    async createOrder (symbol: string, type, side: OrderSide, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name bingx#createOrder
+         * @description create a trade order
+         * @see https://bingx-api.github.io/docs/spot/trade-interface.html#create-an-order
+         * @see https://bingx-api.github.io/docs/swapV2/trade-api.html#_1-trade-order
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} params extra parameters specific to the bingx api endpoint
```

Comment:
@AlgoQ we're missing the stop orders type 2 unification (stopLossPrice, takeProfitPrice) here
![image](https://github.com/ccxt/ccxt/assets/43336371/f7671a87-5cbd-4345-829b-bec3b8fca807)